packages/flow-dev-tools/package.json

```diff
@@ -49,5 +53,8 @@
   },
   "scripts": {
     "test": "make -C ../.. test"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0"
```

Comment:
why do you need babel-cli here? alternately, why did you use the dist version in `tool`?

Reply:
@threepointone tbh, I followed what was done in the flow-upgrade package. I imagine we use some modern syntax that might not be supported out of the box. What do you think is the right way?

Reply:
I dunno :) I was just curious. `babel-node` isn't called anywhere, so wanted to understand why you added this here

Reply:
Oh right. Good catch. I forgot to add the build step 🤦🏻‍♂️Will do.

Reply:
would it be simpler to just run `src/main.js`?

Reply:
We use es6 imports in the package. So I'm not sure how using the source directly would work if we plan on distributing the package.

Reply:
It already uses `require("babel-register")(babelConfig);` so I imagine it would just work? I don't think anyone will `import Something from 'flow-dev-tools'` in their code.

Don't mind me, I'm just doing driveby commenting :)

Reply:
You're right. I misunderstood your initial comment.